player/webpack.config.js

```diff
@@ -251,4 +270,30 @@ module.exports = (env, argv) => {
 
         }
     }
-};
\ No newline at end of file
+};
+
+function updateCordovaConfigXML(channel) {
+    var replaceTo = (channel === 'sunbird') ? APP_CONFIG.sunbird.splashScreen.backgroundImage : APP_CONFIG.ekstep.splashScreen.backgroundImage
+    const options = {
+        files: './config.dist.xml',
+        from: /SPLASH_IMAGE_PATH/g,
+        to: replaceTo,
+    };
+    file_extra.copy('./config.xml', './config.dist.xml')
```

Comment:
Make it generic as source, destination, options.
Options should have source string & replace string